### PR TITLE
Added better card display.

### PR DESCRIPTION
**Pull request by admoore0**

### CODE REVIEW

File: src/states/game/game.gd [hunk 1]
```diff
@@ -273,16 +282,24 @@ func draw_card() -> void:
...
+	for card in card_nodes.get_children() as Array[DualCard]:
+		card.draggable = false
+
 	var dual_card_data = deck.pop_front()
 	hand.append(dual_card_data)
-	var card = preload("res://src/cards/dual_card.tscn").instantiate()
-	card_nodes.add_child(card)
-	card.initialize(dual_card_data)
-	card.dropped_card.connect(self._on_card_dropped)
+	var new_card = preload("res://src/cards/dual_card.tscn").instantiate()
```
Comment (thearst3rd) [inline src/states/game/game.gd]: `:=` 😈

Comment (thearst3rd) [reply src/states/game/game.gd]: i will now commit die

File: src/ui/card_viewer.tscn [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[gd_scene load_steps=2 format=3 uid="uid://j24hlutfi1ip"]
+[gd_scene load_steps=2 format=3 uid="uid://dimjfi53hnnof"]
```
Comment (thearst3rd) [inline src/ui/card_viewer.tscn:1]: lol

**Review by thearst3rd — APPROVED** [review]

**Review by kdreese — APPROVED** [review]

File: src/states/game/game.gd (hunk 1 above)
Comment (admoore0) [reply src/states/game/game.gd]: smh i keep getting blamed for other people's code 😤

**Review by kdreese — APPROVED** [review]

**Pull request merged**